app/assets/stylesheets/index.scss

```diff
@@ -0,0 +1,12 @@
+#welcome {
+  position: relative;
+}
+
+.welcome_text {
+  width: 50%;
+  height: 50%;
+  overflow: auto;
+  margin: auto;
+  position: absolute;
+  top: 0; left: 0; bottom: 0; right: 0;
```

Comment:
Property 'left' should be placed on own line<br>Property 'bottom' should be placed on own line<br>Property 'right' should be placed on own line